src/scene.rs

```diff
@@ -18,6 +25,7 @@ impl Scene {
         Self::default()
     }
 
+    // TODO - Rename to "clear()" (see #469)
```

Comment:
I think this TODO can be removed.

Reply:
@PoignardAzur looks like this TODO is still present.